cfg/error_messages.go

```diff
@@ -42,19 +42,8 @@ func displayWtfConfigDirCreateError(err error) {
 	displayError(err)
 }
 
-func displayWtfConfigFileLoadError(err error) {
-	fmt.Printf("\n%s Could not load '%s'.\n", aurora.Red("ERROR"), aurora.Yellow(WtfConfigFile))
-	fmt.Println()
-	fmt.Println("This could mean one of two things:")
-	fmt.Println()
-	fmt.Printf("    1. Your %s file is missing. Check in %s to see if %s is there.\n", aurora.Yellow(WtfConfigFile), aurora.Yellow("~/.config/wtf/"), aurora.Yellow(WtfConfigFile))
```

Comment:
This was differentiated because in the case of an explicitly-passed in file, the user knows which file the message is talking about.

In the case above, using the default config, the user may not know where the config file lives. This message tells them that.

Reply:
If that can be maintained while getting rid of the `bool` that's passed around, that would be great.

Reply:
Yeah, it works.

With default (triggered by bad yaml):

<img width="780" alt="Screen Shot 2019-09-13 at 7 14 14 PM" src="https://user-images.githubusercontent.com/1523955/64899726-cf032000-d65a-11e9-816b-a6a5deb316bc.png">

File that doesn't exist:

<img width="790" alt="Screen Shot 2019-09-13 at 7 14 31 PM" src="https://user-images.githubusercontent.com/1523955/64899730-d4606a80-d65a-11e9-9605-1499a2b670a7.png">

Reply:
Sweet.